src/Microsoft.ML.Data/Transforms/ValueMapping.cs

```diff
@@ -324,11 +339,14 @@ public class ValueMappingTransformer : OneToOneTransformerBase
 
         // Stream names for the binary idv streams.
         private const string DefaultMapName = "DefaultMap.idv";
-        internal static string KeyColumnName = "Key";
-        internal static string ValueColumnName = "Value";
+        internal static string DefaultKeyColumnName = "Key";
+        internal static string DefaultValueColumnName = "Value";
         private ValueMap _valueMap;
         private readonly byte[] _dataView;
 
+        private readonly DataViewSchema.Column _lookupKeyColumn;
+        private readonly DataViewSchema.Column _lookupValueColumn;
```

Comment:
>private [](start = 8, length = 7)

Why store these? They look like they can be removed.

Reply:
Super nice catch. Thanks.

---
In reply to: [271535842](https://github.com/dotnet/machinelearning/pull/3180#discussion_r271535842) [](ancestors = 271535842)